guidelines.md

```diff
@@ -44,6 +44,20 @@ A unique `AccessToken` will automatically be generated and shared with you for e
 
 All operations of the API optionally accept `LanguageCode` and `CultureCode`. These can be used to enforce language and culture of the operation which affects e.g. names of entities, descriptions or error messages. Both of these values must be defined together otherwise default values of the [Enterprise](operations/configuration.md#enterprise) are used.
 
+### Request limits
```

Comment:
nitpick: I'd go with just Limits beause it's already subsection of Request section.